validation/src/main/scala/hmda/validation/api/ValidationApi.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.validation.api
+
+import hmda.validation.dsl.{ Failure, Result, Success }
+import hmda.validation.engine.ValidationError
+import scalaz._
+import scalaz.Scalaz._
+
+trait ValidationApi {
+
+  def convertResult[T](input: T, result: Result, ruleName: String): ValidationNel[ValidationError, T] = {
+    result match {
+      case Success() => input.success
+      case Failure(msg) => ValidationError(s"$ruleName  failed: $msg").failure.toValidationNel
+    }
+  }
+
+  def validateAll[E, T](checks: List[ValidationNel[E, T]], t: T): ValidationNel[E, T] = {
```

Comment:
Now that this method is no longer in the Transmittal Sheet context, `t` is confusing as a variable name. Could `t` be renamed to `input`?